README.md

```diff
@@ -35,7 +35,7 @@ information about its parameters, by running `python comp_loinc/build.py COMMAND
 `python comp_loinc/build.py merge --owl-directory data/output/owl_component_files/ --output data/output/merged_loinc.owl`
 
 ### 1.5. `reason`: Run the reasoner using elk to create the composed code classes
-`python comp_loinc/build.py reason --merged-owl data/output/merged_loinc.owl --owl_reasoner elk --output data/output/merged_reasoned_loinc.owl`
```

Comment:
I encountered this error and didn't see it!

Reply:
@putmantime I think I found out what the error is. One of the final steps I didn't do was try running it from the `makefile`. But also running `python3 - m unittest discover` manually results in the same error.

The issue is that I wanted to make the test "more robust" and have it call the CLI, rather than call the Python API (the ingest functions) directly. However, this involves using a subprocess, and the interpreter in the subprocess could be a different Python interpreter.

At first, it was selecting python2 on my computer. I fixed that. But now it's still selecting some interpreter that doesn't have anything `pip installed` on it yet. It would be really annoying to have it do that every time subprocess is called, but I might try that. That could be the quick fix. But otherwise another fix would be to manually pass the interpreter path, or some other more complex solution.

I'll probably call the Python API directly instead though. Quicker and then update these tests later.